### ISSUE TITLE
Tempest Double + Focus bug

### ISSUE THREAD

**Issue by Draze22** [arcadia-redux/custom_hero_clash_issues#1232]
![image](https://user-images.githubusercontent.com/63279022/120270909-a8cd5a00-c2ed-11eb-9e14-fd6f6b97bf8e.png)

**Comment by efe-efe** [comment]
I fixed the buge mention but found a new one which maybe needs its own issue: When you get the Focus innate before learning Tempest Double, the clone will never get the focus buff. If you do it the other way around, it works fine

**Comment by SanctusAnimus** [comment]
> When you get the Focus innate before learning Tempest Double  

Is that a thing even without PR you've opened? Even if it's not caused by it, might as well fix it in it, since it's called Tempest Focus already

**Comment by efe-efe** [comment]
> > When you get the Focus innate before learning Tempest Double, the clone will never get the focus buff
> 
> Is that a thing even without PR you've opened? Even if it's not caused by it, might as well fix it in it, since it's called Tempest Focus already

Yeah indeed i found that bug before starting to work on the issue. I can try to fix it inside the same PR. I'll let you know when it is ready 👍

**Comment by efe-efe** [comment]
@SanctusAnimus, hey, it took me more than expected to find the source of the second bug, but i finally fixed it, so it is ready for the code review 👌
